### DISCUSSION TITLE
Motif enrichment analysis in promoter and non-promoter regions

### DISCUSSION THREAD

**Discussion by bj-chen** [aertslab/scenicplus#500] [Q&A]
Dear SCENIC+ Development Team,

Thank you for providing the community with such a valuable tool. I have a question and would appreciate your insights. In your paper published in Nature Methods, you mention that motif enrichment analysis is conducted both with and without including promoters.

Could you please clarify how SCENIC+ handles the results of motif enrichment in promoters versus non-promoters? i.e are they merged or used separately at later stages? Is the information regarding motif enrichment in these two categories reflected in the final outputs?

Thank you very much for your assistance.

Best regards,
Bei Jun

**Comment by SeppeDeWinter** [comment]
Hi @bj-chen 

This is the same kind of motif enrichment analysis, the only difference is the input region set. In one case it includes promoters in another case it does not.

The results of both analysis are indeed merged at a later stage.

All the best,

S
